dashboard/agent.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         runtime_env_dir: str,
         logging_params: dict,
         agent_id: int,
+        cluster_id: str,
```

Comment:
Instead of pasring session_dir I decided to pass the cluster id because that seems to be cleaner (and it has less coupling between session name & cluster id)